app/code/Magento/Braintree/Model/Adminhtml/Source/PaymentAction.php

```diff
@@ -22,11 +21,11 @@ public function toOptionArray()
     {
         return [
             [
-                'value' => AbstractMethod::ACTION_AUTHORIZE,
+                'value' => 'authorize',
```

Comment:
I don't see any objections of `string` usage.

Reply:
Using constant where it possible allow you to find all places where this constant used just with "find usages" in any IDE. In Magento 1.x we had a lot of such places where were used strings instead of constants and it was really chaos.

Reply:
Agree, but in this case these constants from the deprecated class. The better solution to have such constants in the `\Magento\Payment\Model\MethodInterface` but now I can't introduce them - it will be BIC.

Reply:
I checked, these constants are still used in paypal, auth.net payment methods, sales module. For sure these constants were used in much more places if we will look in custom modules, so this class will live marked as "deprecated" really long time.

In order to fix it - we should re-add these constants to new or existing class/interface that is not marked as deprecated.

I think such changes should be discussed with magento architects.

Reply:
@ihor-sviziev I've added them to `MethodInterface` in the PR for 2.3-develop